### PR TITLE
REV: workbench user preferences menu bar

### PR DESCRIPTION
**Pull request by bxf12315**
@manstis  @Salaboy 
Please review it.

### CODE REVIEW

**Comment by manstis** [comment]
Looks OK to me - other than some formatting issues (but I can fix when merging).

@Salaboy OK with you?

**Comment by bxf12315** [comment]
@manstis  what's formatting issues? 
i can fix it. but i was used all drools format.

**Comment by manstis** [comment]
@bxf12315 just some cosmetic stuff, 

Extra white space: e.g. https://github.com/droolsjbpm/kie-wb-common/pull/17/files#diff-94eb83628535180d1d01c2eb6ba4efdfR141 and https://github.com/droolsjbpm/kie-wb-common/pull/17/files#diff-c0a67820bb317c9ee9d091e8d9889b99R12

Consistent use of "final" on parameters (e.g. https://github.com/droolsjbpm/kie-wb-common/pull/17/files#diff-94eb83628535180d1d01c2eb6ba4efdfR195 uses "final" whereas https://github.com/droolsjbpm/kie-wb-common/pull/17/files#diff-94eb83628535180d1d01c2eb6ba4efdfR186 does not).

Header comment needs formatting: https://github.com/droolsjbpm/kie-wb-common/pull/17/files#diff-6c108cc97bc3a2662ab9e2bc7f73d803R3

There's more.. just scan the files and you should see.

**Comment by bxf12315** [comment]
I will fix these formatter issue and raise new PR

File: kie-wb-common-widgets/kie-wb-common-ui/src/main/java/org/kie/workbench/common/widgets/client/popups/workbench/configuration/WorkbenchConfigurationPopup.java [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+import com.google.gwt.uibinder.client.UiBinder;
+import com.google.gwt.uibinder.client.UiField;
+import com.google.gwt.user.client.Command;
+import com.google.gwt.user.client.Window;
+import com.google.gwt.user.client.ui.Widget;
+
+@Dependent
+public class WorkbenchConfigurationPopup extends BaseModal {
+
+    private static AddNewKBasePopupViewImplBinder uiBinder = GWT
```
Comment (Salaboy) [inline kie-wb-common-widgets/kie-wb-common-ui/src/main/java/org/kie/workbench/common/widgets/client/popups/workbench/configuration/WorkbenchConfigurationPopup.java]: This is wrong right?

File: kie-wb-common-widgets/kie-wb-common-ui/src/main/java/org/kie/workbench/common/widgets/client/popups/workbench/configuration/WorkbenchConfigurationPopup.java [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+            .create( AddNewKBasePopupViewImplBinder.class );
```
Comment (Salaboy) [inline kie-wb-common-widgets/kie-wb-common-ui/src/main/java/org/kie/workbench/common/widgets/client/popups/workbench/configuration/WorkbenchConfigurationPopup.java]: This one as well

**Pull request closed without merging**